tools/benchmark/src/mocha/memoryTestRunner.ts

```diff
@@ -24,6 +24,8 @@ import { timer } from "../timer";
 
 import { supportParentProcess } from "./runner";
 
+const ENABLE_MEM_REGRESSION = process.env.ENABLE_MEM_REGRESSION === "true";
```

Comment:
If the env variable is necessary for the regression-testing behavior, let's document that in the relevant properties, otherwise people could get confused as to why their tests are not regressing.

I'm also on the fence about the need for the env variable... granted, it gives us flexibility to run them and not fail things, but then when/how do we plan to run tests with this variable set?